streamingpro-spark-2.2.0-adaptor/src/main/java/org/apache/spark/MLSQLConf.scala

```diff
@@ -200,6 +200,24 @@ object MLSQLConf {
       .longConf
       .createWithDefault(1000)
 
+  val MLSQL_CACHE_PASSIVE_AUTO_CLEANUP_ENABLE: ConfigEntry[Boolean] =
+    MLSQLConfigBuilder("spark.mlsql.sql.cache.passive.cleanup.enable")
```

Comment:
spark.mlsql is used in spark conf; If you want to use in MLSQL, you should start with streaming prefix.